src/couch/src/couch_att.erl

```diff
@@ -680,6 +691,22 @@ upgrade_encoding(false) -> identity;
 upgrade_encoding(Encoding) -> Encoding.
 
 
+max_attachment_size() ->
+    case config:get("couchdb", "max_attachment_size", "infinity") of
```

Comment:
Side note: we probably should just made `config:get_integer/3` to believe in `infinity`, it's not a first time I'm seeing this dance.

Reply:
True, there enough place we do this that we could probably `get_integer_or_infinity`